rules/linux/lnx_system_net_disc_firewall_enum.yml

```diff
@@ -0,0 +1,31 @@
+title: System Network Discovery - Firewall Enumeration
+id: 71da9e5a-fb1e-46a8-abc1-28c80173af4c
+status: experimental
+description: Detects enumeration of firewall configuration
+author: remotephone, oscd.community
+date: 2020/10/06
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1016/T1016.md
+    - https://attack.mitre.org/techniques/T1016
+logsource:
+    product: unix
```

Comment:
I would rather use macOS category! (your rule is going to be the first one! congratulations!)
this way you better create multiple rules for different platforms.

Reply:
Done, thank you! These are the first rules I've written in sigma so that's exciting!

Reply:
you did a great job, @remotephone!